CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+* Update high resolution media query
```

Comment:
We will need the `### HEAD` thing [like we did before](https://github.com/h5bp/html5-boilerplate/commit/1c93f0a0569231c820013042c2c19102a32d1693#diff-4ac32a78649ca5bdd8e0ba38b7006a1eL1).